v2/anomalydetection/backend-setup.mdx

```diff
@@ -0,0 +1,1915 @@
+
+        request_id = request_body.get("requestId")
+        if not request_id:
+            return GeneralErrorResponse(message="The request ID is required")
+
+        action_type = 'passwordless-send-sms'
+        ip = get_ip_from_request(api_options.request)
+        email = None
+        phone_number = None
+        codes = await list_codes_by_device_id(tenant_id=tenant_id, device_id=device_id, user_context=user_context)
+        if codes is not None:
+            email = codes.email
+            phone_number = codes.phone_number
+        identifier = None
+        if email is not None:
+            identifier = email
+        elif phone_number is not None:
+            identifier = phone_number
+        brute_force_config = get_brute_force_config(identifier, ip, action_type)
+        
+        # we check the anomaly detection service before calling the original implementation of signUp
+        security_check_response = await handle_security_checks(
+            request_id=request_id,
+            password=None,
+            brute_force_config=brute_force_config,
+            email=email,
+            phone_number=phone_number,
+            action_type=action_type
+        )
+        if security_check_response is not None:
+            return security_check_response
+
+        # We need to call the original implementation of sign_up_post.
+        response = await original_resend_code_post(device_id, pre_auth_session_id, tenant_id, api_options, user_context)
+
+        return response
+    original_implementation.resend_code_post = resend_code_post
+
+    return original_implementation
+# highlight-end
+
+
+init(
+    app_info=InputAppInfo(api_domain="...", app_name="...", website_domain="..."),
+    framework='...', # type: ignore
+    recipe_list=[
+        passwordless.init(
+            # highlight-start
+            flow_type="USER_INPUT_CODE_AND_MAGIC_LINK",
+            contact_config=passwordless.ContactEmailOrPhoneConfig(),
+            override=passwordless.InputOverrideConfig(
+                apis=override_passwordless_apis
+            )
+            # highlight-end
+        )
+    ]
+)
+```
+
+</TabItem>
+
+<TabItem value="otherFrameworks">
+other frameworks
+</TabItem>
+</BackendSDKTabs>
+
+
+### Third party
+
+It's important to note that anomaly detection is not recommended for use with third-party providers. There are several reasons for this:
+
+1. **Existing anomaly detection**: Most reputable third-party authentication providers (like Google, Facebook, Apple, etc.) have robust security measures in place, including their own anomaly detection systems. These systems are typically more comprehensive and tailored to their specific platforms.
+
+2. **Cost considerations**: Implementing additional anomaly detection on top of third-party authentication can lead to unnecessary costs. You'd essentially be paying for a service that's already being provided by the third-party, resulting in redundant expenditure.
```

Comment:
```suggestion
```

Reply:
We charge users the same even if they are not using for third party, but for emailpassword for example

Reply:
done